developer/src/kmc/src/commands/buildTestData/index.ts

```diff
@@ -21,10 +21,10 @@ export async function buildTestData(infile: string, _options: any, commander: an
     }
   };
 
-  const callbacks: CompilerCallbacks = new NodeCompilerCallbacks(options);
+  const callbacks: NodeCompilerCallbacks = new NodeCompilerCallbacks(options);
```

Comment:
```suggestion
  const callbacks = new NodeCompilerCallbacks(options);
```

Type should be inferred I think?

Reply:
Yes… Needs the subclass to make this call.

Reply:
But shouldn't need to include the type, because `const x = new foo()` gives `x` the type `foo` doesn't it?

Reply:
fixed